node/impl/full/eth.go

```diff
@@ -1612,6 +1623,13 @@ func ethGasSearch(
 }
 
 func (a *EthModule) EthCall(ctx context.Context, tx ethtypes.EthCall, blkParam ethtypes.EthBlockNumberOrHash) (ethtypes.EthBytes, error) {
```

Comment:
Where are we returning an `ExecutionRevertedError` in this function ?

Reply:
@aarshkshah1992 
We are returning the `ExecutionRevertedError` from `applyMessage` function directly, so we don't to do it separately.

In `EstimateGas` we are checking separately because of `GasEstimateMessageGas` function also returns an error.

https://github.com/virajbhartiya/lotus/blob/b1c146a3a2f2f9ca0a1c56a7f0ef7bf438a7db59/node/impl/full/eth.go#L1435